subsystememu/src/Subsystem.scala

```diff
@@ -72,7 +78,7 @@ class VerdesConfig
       .orElse(new WithCacheBlockBytes(16))
       // SoC
       .orElse(new WithoutTLMonitors)
-      .orElse(new WithDebugSBA)
+      .orElse(new WithNExtTopInterrupts(1))
```

Comment:
do we need it?

Reply:
This is useful if we need UART input without polling for debug purposes.